<p align="center">
  <a href="https://grace.biohpc.swmed.edu/">
    <img height="100" src="https://raw.githubusercontent.com/cailing20/GRACE-Web-application/master/images/GraceLogo.png">
  </a>
  <h1>Genomic Regression Analysis of Coordinated Expression</h1>
</p>

Co-expression analysis is widely used to predict gene function and to identify functionally related gene sets. However, co-expression analysis using human cancer transcriptomic data is confounded by somatic copy number alterations (SCNA), which produce co-expression signatures based on physical proximity rather than biological function. To better understand gene-gene co-expression based on biological regulation but not SCNA, we describe a method termed "Genomic Regression Analysis of Coordinated Expression" (GRACE) to adjust for the effect of SCNA in co-expression analysis. The results from analyses of TCGA, CCLE, and NCI60 data sets show that GRACE can improve our understanding of how a transcriptional network is re-wired in cancer. A user-friendly [web database](https://grace.biohpc.swmed.edu/) populated with data sets from The Cancer Genome Atlas (TCGA) is provided to allow customized query.

To use this tool, please go to [https://grace.biohpc.swmed.edu/](https://grace.biohpc.swmed.edu/)

Citation
--------

Ling Cai, Qiwei Li, Yi Du, Jonghyun Yun, Yang Xie, Ralph J. DeBerardinis & Guanghua Xiao  
[Genomic regression analysis of coordinated expression](https://www.nature.com/articles/s41467-017-02181-0)    
Nat Commun. 2017 Dec 19;8(1):2187.
